lxd/resources/utils.go

```diff
@@ -210,3 +215,21 @@ func usbAddress(devicePath string) (string, error) {
 		return fmt.Sprintf("%d:%d", bus, dev), nil
 	}
 }
+
+func getDeviceDir(devicePath string) (string, error) {
```

Comment:
@skatsaounis please can you add a comment explaining what this function does without needing to look at the implementation. Thanks

Reply:
done